### ISSUE TITLE
Replace approvals with linting for certain files

### ISSUE THREAD

**Issue by jpmedley** [GoogleChrome/developer.chrome.com#1895]
⚠️ This is for feature requests with the https://developer.chrome.com site, not Chromium itself.
If you want to file a bug with Chromium (the open-source project behind Google Chrome and other browsers), please use https://crbug.com.

**Is your feature request related to a problem? Please describe.**
The site depends several files with a known and regular structure. Particular examples are the `authorsData.json`, `authors.yaml`, and `tags.yaml` are examples, but there may be others. These files are often updated in the same PRs as new articles, requiring additional reviews by engineers. The engineers are simply reviewing for validity and well-formedness.

**Describe the solution you'd like**
An automated process should lint these files.

**Describe alternatives you've considered**
Authors should submit updates to metafiles as separate PRs. Outside contributors [usually don't know this](https://github.com/GoogleChrome/developer.chrome.com/pull/1862/files). Since outside contributors are doing something outside their normal duties (volunteers essentially), I would personally prefer not to ask for extra work. Even submitting a separating a separate PR myself can sometimes require asking volunteers to remove files from their own PRs.

**Additional context**
There is [an open PR](https://github.com/GoogleChrome/developer.chrome.com/pull/1862/files) that needs to be merged before Chrome 98 goes out later this week. Being on the back end of a holiday, technical reviews may be hard to get in that time. There are similar time throughout the year when such a problem may arise. The separate PR solution explained above, does not solve this problem. The technical reviewers would still be backlogged after a long break or holiday.

**Comment by rachelandrew** [comment]
@jpmedley On web.dev we don't tend to request infra approval for changes to yaml files. Is there something different here that we need to add to this linter in your opinion?

I'd like the two sites to be similar in terms of linter rules and so on.

**Comment by jpmedley** [comment]
I agree the two sites should be similar. I was trying to offer something constructive for the problem [in my linked PR](https://github.com/GoogleChrome/developer.chrome.com/pull/1862/files), namely that an automated process on web.dev requires either @devnook or @MichaelSolati to approve the PR. The PR needs to be merged early tomorrow. As things currently stand, I may need to use my admin power to override the lack of eng approval.

My preference would be that both sites lint so that we avoid breaking author information, but do not need eng approvals for what is essentially information (as opposed to functionality).

**Comment by rachelandrew** [comment]
ok so it looks like this is a review bot issue as we don't require eng approval on web.dev for changes like this, I'll see what I can figure out

**Comment by devnook** [comment]
This is solved by the new reviewbot setup.
